src/visitor/mysql.rs

```diff
@@ -475,6 +475,18 @@ impl<'a> Visitor<'a> for Mysql<'a> {
         Ok(())
     }
 
+    #[cfg(any(feature = "postgresql", feature = "mysql"))]
```

Comment:
Do you really need these feature flags? The whole mysql struct is gated by the `mysql` feature flag already.